examples/mint-repl/mint-repl.rs

```diff
@@ -85,7 +85,7 @@ fn main() -> Result<()> {
     loop {
         match rl.readline(">> ") {
             Ok(line) => {
-                let mut args = line.trim().split_whitespace();
+                let mut args = line.split_whitespace();
```

Comment:
Does this still work when we have leading whitespace in the line input? ie. `<space>reissue`

Reply:
It's supposed to, it's the linter who suggested it: https://rust-lang.github.io/rust-clippy/master/index.html#trim_split_whitespace

Reply:
:+1: nice